<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncodingDialog</class>
 <widget class="QDialog" name="EncodingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Encoding</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="QPushButton" name="reopenButton">
       <property name="text">
        <string>Reopen</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="selectButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QListView" name="encodingListView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLineEdit" name="encodingFilter"/>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>encodingFilter</tabstop>
  <tabstop>encodingListView</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>reopenButton</tabstop>
  <tabstop>selectButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
